examples/demo/src/ewf_roc_server_execution.e

```diff
@@ -92,6 +92,9 @@ feature -- CMS setup
 
 			create {CMS_DEMO_MODULE} m.make
 			a_setup.register_module (m)
+
+			create {GOOGLE_CUSTOM_SEARCH_MODULE} m.make
```

Comment:
I agree, done, maybe I will rename the name of the module to

CMS_GOOGLE_CUSTOM_SEARCH_MODULE, to follow the prefix convention